account_journal_lock_date/wizards/wizard_lock_account_journal.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2017 ACSONE SA/NV
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api, _, SUPERUSER_ID
+
+from ..exceptions import JournalLockDateError
+
+
+class WizardLockAccountJournal(models.TransientModel):
+    _name = "wizard.lock.account.journal"
+    _description = "Lock Account Journal"
+
+    company_id = fields.Many2one(
+        comodel_name='res.company', string="Company", required=True,
+        default=lambda self: self.env.user.company_id)
+    journal_ids = fields.Many2many(
+        'account.journal', rel='wizard_lock_account_move_journal',
+        string='Journals', required=True)
+    lock_date = fields.Date(string='Lock Date', required=True)
+    permanent_lock = fields.Boolean('Permanent Lock')
+
+    @api.multi
+    def _check_execute_allowed(self):
+        self.ensure_one()
+        obj_move = self.env['account.move']
+        has_adviser_group = self.env.user.has_group(
+            'account.group_account_manager')
+        if not (has_adviser_group or self.env.uid == SUPERUSER_ID):
+            raise JournalLockDateError(
+                _("You are not allowed to execute this action."))
```

Comment:
This part about access rights should be kept isolated in a separate method, for easy override in customer projects.